metapool/count.py

```diff
@@ -136,22 +136,35 @@ def _safe_get(_document, _key):
         return _document[_key]
 
 
-def bcl2fastq_counts(run_dir, sample_sheet):
+def raw_read_counts(run_dir, sample_sheet):
     bcl2fastq_path = join(abspath(run_dir), 'Stats/Stats.json')
     bclconvert_path = join(abspath(run_dir), 'Reports/Demultiplex_Stats.csv')
+    seqcounts_path = join(abspath(run_dir), 'Reports/SeqCounts.csv')
 
-    if exists(bcl2fastq_path):
-        if exists(bclconvert_path):
-            raise IOError(f"both '{bcl2fastq_path}' and '{bclconvert_path}'"
-                          " exist")
-        else:
-            return _bcl2fastq_counts(bcl2fastq_path)
-    elif exists(bclconvert_path):
+    count = 0
+    found = []
+    for item in [bcl2fastq_path, bclconvert_path, seqcounts_path]:
+        if exists(item):
+            count += 1
+            found.append(item)
+
+        if count > 1:
+            raise IOError("multiple metadata files exist: %s",
+                          ', '.join(found))
+
+    if len(found) == 0:
+        raise IOError(f"Cannot find Stats.json '{bcl2fastq_path}' or "
+                      f"Demultiplex_Stats.csv '{bclconvert_path}' or "
+                      f"SeqCounts.csv '{seqcounts_path}' for this run")
+
+    if found[0] == bcl2fastq_path:
+        return _bcl2fastq_counts(bcl2fastq_path)
+    elif found[0] == bclconvert_path:
         return _bclconvert_counts(bclconvert_path)
+    elif found[0] == seqcounts_path:
+        return _seqcount_counts(seqcounts_path)
     else:
-        raise IOError(f"Cannot find Stats.json '{bcl2fastq_path}' or "
-                      f"Demultiplex_Stats.csv '{bclconvert_path}' for this"
-                      " run")
+        raise IOError("Undetermiend Error")
```

Comment:
typo in "Undetermined" :). May I suggest: 
`raise IOError(f"Undetermined error accessing {found[0]}")`?